imgee/tasks.py

```diff
@@ -0,0 +1,35 @@
+import redis
+from imgee import app
+
+
+class TaskRegistry(object):
+    def __init__(self, name='default', connection=None):
+        self.connection = redis.from_url(connection) if connection else None
+        self.name = name
+        self.key = 'imgee:registry:%s' % name
+
+    def set_connection(self, connection=None):
+        connection = connection or app.config.get('REDIS_URL')
+        self.connection = redis.from_url(connection)
+
+    def add(self, taskid):
+        self.connection.sadd(self.key, taskid)
```

Comment:
Can we add a timestamp along with this key and then clear that ourselves inside the `get_resized_image()` function?